src/components/widgets/InputWidget.js

```diff
@@ -32,10 +51,13 @@ export default class InputWidget extends React.Component {
     return (
       <input
         id={this.props.id}
+        ref={this.input}
```

Comment:
I think you need to add "this.input = React.createRef();" to the constructor of input widget, otherwise this ref prop will not get referenced